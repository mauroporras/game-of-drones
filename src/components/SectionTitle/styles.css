.SectionTitle {
  color: var(--color-black);
  margin: 0 0 1rem;
  text-align: center;
}
