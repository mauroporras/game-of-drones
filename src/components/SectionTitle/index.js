import React from 'react'

import './styles.css'

const SectionTitle = props => (
  <h3 className='SectionTitle'>{props.text}</h3>
)

export default SectionTitle
