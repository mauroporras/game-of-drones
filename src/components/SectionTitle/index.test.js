import React from 'react'
import renderer from 'react-test-renderer'

import SectionTitle from './index'

test("Component renders the correct text", () => {
  const component = renderer.create(
    <SectionTitle text='Hello' />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
