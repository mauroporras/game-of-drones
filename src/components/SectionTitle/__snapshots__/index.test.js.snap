exports[`test Component renders the correct text 1`] = `
<h3
  className="SectionTitle">
  Hello
</h3>
`;
